src/CoreWCF.ConfigurationManager/src/CoreWCF/Configuration/ServiceDefaults.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace CoreWCF.Configuration
+{
+    static class ServiceDefaults
+    {
```

Comment:
There are several places where we have missing accessors (internal in this case). If you place the cursor in the word static, press ctrl + ., choose "Add acessibility modifiers" and select to fix it in the entire project, you should be able to fix them all in one go.